Library/Homebrew/env_config.rb

```diff
@@ -5,166 +5,171 @@ module EnvConfig
         boolean:     true,
       },
-      HOMEBREW_FORCE_BREWED_GIT:          {
+      HOMEBREW_FORCE_BREWED_GIT:             {
         description: "If set, always use a Homebrew-installed `git`(1) rather than the system version. " \
                      "Automatically set if the system version of `git` is too old.",
         boolean:     true,
       },
-      HOMEBREW_FORCE_HOMEBREW_ON_LINUX:   {
+      HOMEBREW_FORCE_HOMEBREW_ON_LINUX:      {
         description: "If set, running Homebrew on Linux will use URLs for macOS. This is useful when merging " \
                      "pull requests for macOS while on Linux.",
         boolean:     true,
       },
-      HOMEBREW_FORCE_VENDOR_RUBY:         {
+      HOMEBREW_FORCE_VENDOR_RUBY:            {
         description: "If set, always use Homebrew's vendored, relocatable Ruby version even if the system version " \
                      "of Ruby is new enough.",
         boolean:     true,
       },
-      HOMEBREW_GITHUB_API_PASSWORD:       {
+      HOMEBREW_GITHUB_API_PASSWORD:          {
         description: "Use this password for authentication with the GitHub API, for features " \
                      "such as `brew search`. We strongly recommend using `HOMEBREW_GITHUB_API_TOKEN` instead.",
       },
-      HOMEBREW_GITHUB_API_TOKEN:          {
+      HOMEBREW_GITHUB_API_TOKEN:             {
         description: "Use this personal access token for the GitHub API, for features such as " \
                      "`brew search`. You can create one at <https://github.com/settings/tokens>. If set, " \
                      "GitHub will allow you a greater number of API requests. For more information, see: " \
                      "<https://developer.github.com/v3/#rate-limiting>\n\n    *Note:* Homebrew doesn't " \
                      "require permissions for any of the scopes, but some developer commands may require " \
                      "additional permissions.",
       },
-      HOMEBREW_GITHUB_API_USERNAME:       {
+      HOMEBREW_GITHUB_API_USERNAME:          {
         description: "Use this username for authentication with the GitHub API, for features " \
                      "such as `brew search`. We strongly recommend using `HOMEBREW_GITHUB_API_TOKEN` instead.",
       },
-      HOMEBREW_GIT_EMAIL:                 {
+      HOMEBREW_GIT_EMAIL:                    {
         description: "Set the Git author and committer name to this value.",
       },
-      HOMEBREW_GIT_NAME:                  {
+      HOMEBREW_GIT_NAME:                     {
         description: "Set the Git author and committer email to this value.",
       },
-      HOMEBREW_INSTALL_BADGE:             {
+      HOMEBREW_INSTALL_BADGE:                {
         description:  "Print this text before the installation summary of each successful build.",
         default_text: 'The "Beer Mug" emoji.',
-        default:      "🍺",
+        default:      "Ã°ÂÂÂº",
```

Comment:
Something went wrong

Reply:
ah, that's `brew style --fix`. I didn't touch here anything besides lines 94-98